src/Services/ConsignmentEncode.php

```diff
@@ -84,6 +84,7 @@ public static function encodeExtraOptions(array $consignmentEncoded, AbstractCon
                     'only_recipient'    => (int) $consignment->isOnlyRecipient(),
```

Comment:
Misschien filterNull (die hebben we als het goed is ergens) hier omheen zetten, dan sloopt het ook niet de api